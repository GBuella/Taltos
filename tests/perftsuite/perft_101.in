setboard k7/8/6p1/8/8/7P/8/K7 b - - 0 1
echo k7/8/6p1/8/8/7P/8/K7 b - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
