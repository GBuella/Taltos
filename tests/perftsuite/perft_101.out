k7/8/6p1/8/8/7P/8/K7 b - - 0 1
4
16
101
637
4354
29679
